README.md

```diff
@@ -1458,7 +1458,8 @@ Enable the Autopause functionality by setting:
 ```
 
 Autopause is not compatible with `EXEC_DIRECTLY=true` and the two cannot be set together.
-
+You should use mc-health instead mc-status for healtcheck/readiness probe with autopause enabled. mc-status is known to have issues with paused servers.
```

Comment:
I don't like the negative wording of this. mc-health is simply a wrapper of mc-status that is autopause aware and should be used for health checks:

https://github.com/itzg/docker-minecraft-server/blob/39519427bbc638442773329d88a8bd2d35c42691/bin/mc-health#L12

Reply:
Yeah I get your point. What would be your suggestion to reword that? Can't think of "neutral" way of saying that.

Reply:
> When configuring kubernetes readiness/liveness health checks with auto-pause enabled, be sure to reference the `mc-health` wrapper script rather than `mc-status` directly.